nextflow.config

```diff
@@ -213,7 +213,7 @@ manifest {
     description     = """Arborator: Genomic Profile Clustering and Summary"""
     mainScript      = 'main.nf'
     nextflowVersion = '!>=23.04.0'
-    version         = '0.1.0'
+    version         = '0.3.0'
```

Comment:
This should be version `0.2.0`

Reply:
🤦‍♀️ Thank you for catching this! Fixed here: [288ce4d](https://github.com/phac-nml/arboratornf/pull/19/commits/288ce4d1619b9b55aada24b152507466cc4cc12d)